flycheck.el

```diff
@@ -7182,7 +7182,7 @@ See Info Node `(elisp)Byte Compilation'."
 
 Variables are taken from `flycheck-emacs-lisp-checkdoc-variables'."
   `(progn
-     ,@(seq-map (lambda (opt) `(setq-default ,opt ,(symbol-value opt)))
+     ,@(seq-map (lambda (opt) `(setq-default ,opt ,(custom-quote (symbol-value opt))))
```

Comment:
Sorry to turn this into a long thread :/ I'm not sure what your example shows...

```
ELISP> (if 'nil false t)
t
ELISP> (equal (if 'nil false t) (if nil false t))
t
``